larreco/RecoAlg/TrackMomentumCalculator.cxx

```diff
@@ -120,6 +183,103 @@ namespace {
     std::vector<double> const xmeas_;
     std::vector<double> const ymeas_;
     std::vector<double> const eymeas_;
+    double const correction_;
+  };
+
+  class FcnWrapperLLHD {
+  public:
+    explicit FcnWrapperLLHD(std::vector<double>& dEi,
+                            std::vector<double>& dEj,
+                            std::vector<double>& dthij,
+                            std::vector<double>& ind,
+                            std::vector<bool>& dthij_valid,
```

Comment:
Suggest changing each of these arguments to r-value references: `std::vector<double>&` -> `std::vector<double>&&`.